project/plugins.sbt

```diff
@@ -1,13 +1,15 @@
-resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.gov.uk/maven2"
+resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
 resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 resolvers += Resolver.typesafeRepo("releases")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.6.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.8.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.15")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.7")
```

Comment:
Guessing it was a typo to downgrade the play version? The latest is "2.8.17"

Reply:
this is not type this is syncing version from https://github.com/hmrc/estates-store/blob/main/project/AppDependencies.scala#L8 inside transitive dependency you can see version in this variable https://github.com/hmrc/estates-store/blob/main/project/AppDependencies.scala#L1

Reply:
I'm still not sure why we'd want to downgrade the version though, isn't it better to override the lower transitive version with the newer version that is working fine?

Reply:
the problem is that you are not overriding library version here but only updating plugin version plugin is nothing but a bunch of config classes and other support stuff for sbt, in this you/we are creating problem as you are pulling new version of plugin and not the library so you might try to build or override a property at compile time but it is not working because new version of plugin is using new version of property from the library but because we cant update library as it is coming from bootstrap we are stuck with incompatible version, so solution is to set the same version as in the AppDependency for compatibility reasons. It is just luck that two completely different version are working now, if I will update to latest .17 it might stop working at all, does that make sense?

Reply:
I'm not sure if I'm misunderstanding something but I feel like the plugin must bring in the libraries? I just made a new test project using the default seed command `sbt new playframework/play-scala-seed.g8` and it doesn't add anything to the libraryDependencies (apart from guice) in the build.sbt, just enables the plugin yet it still brings in all the play dependencies when I check the dependency tree

Reply:
ok then I misunderstood based on their github page if that is the case I will revert this change